.pylintrc

```diff
@@ -0,0 +1,31 @@
+[GENERAL]
+ignore-paths=
+    .git,
+    __pycache__,
+    .pytest_cache,
+    .mypy_cache,
+    .venv,
+    .vscode,
+    .idea,
+    test
+
+ignore=*.md
+
+[MESSAGES CONTROL]
+disable=all
+enable=
+    R0913
+    C0302, # too many lines
+    R0915, # too many statements
+    R0101, # nested blocks level
+    R0801 # duplicate code
```

Comment:
This seems to be a very narrow list of linting targets.

Reply:
Is is a list of clean code related rules that have significant impact on quality, maintainability, understanding and testability.
It is better than the 0 we currently have.
If we don't start small, we will be inundated with many errors most of which have low impact and value.
As we go along, we shall add more rules.